ctapipe/coordinates/tests/test_coordinates.py

```diff
@@ -9,13 +13,81 @@ def test_cam_to_nominal():
     telescope_pointing = SkyCoord(alt=70 * u.deg, az=0 * u.deg, frame=HorizonFrame())
     array_pointing = SkyCoord(alt=72 * u.deg, az=0 * u.deg, frame=HorizonFrame())
 
-    cam_frame = CameraFrame(focal_length=28 * u.m, pointing_direction=telescope_pointing)
+    cam_frame = CameraFrame(focal_length=28 * u.m, telescope_pointing=telescope_pointing)
     cam = SkyCoord(x=0.5 * u.m, y=0.1 * u.m, frame=cam_frame)
 
-    nom_frame = NominalFrame(array_direction=array_pointing)
+    nom_frame = NominalFrame(origin=array_pointing)
     cam.transform_to(nom_frame)
 
 
+def test_icrs_to_camera():
+    from ctapipe.coordinates import CameraFrame, HorizonFrame
+
+    obstime = Time('2013-11-01T03:00')
+    location = EarthLocation.of_site('Roque de los Muchachos')
```

Comment:
Does this test need an internet connection? Or is `of_site` also working offline?